__tests__/app.test.js

```diff
@@ -70,3 +71,16 @@ describe("/api/topics", () => {
     });
   });
 });
+describe("/api", () => {
+  test("GET 200: responds with 200 status", () => {
+    return request(app).get("/api").expect(200);
+  });
+  test("GET 200: responds with object with property of endpoints which contains the contents of endpoints.json file", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
```

Comment:
If you're splitting it down into two tests, with the first test asserting that the status code is `200`, then you can remove line 81 which is doing the same thing in this second test 👍